static_report/cypress/e2e/comparison-report.cy.ts

```diff
@@ -49,15 +49,6 @@ describe('Comparison Report', () => {
     cy.url().should('equal', 'http://localhost:3001/#/');
   });
 
```

Comment:
Add test case?:
Would it be valuable to test the collapse content function here? 
I'm thinking about scenario where something is off with the data and user cannot expand/collapse.

Reply:
When the assertion is empty, it cannot expand/collapse, but our e2e test data didn't cover this.

I added a prop to check whether the assertion exists or not and decide whether the content is collapseable.

Reply:
I feel like that prop isn't needed, as we should avoid adding test "props" in our components. 
I was more thinking that we can check the height value, which is what chakra uses 
![image](https://user-images.githubusercontent.com/11186480/187841017-78687f11-3638-44e3-bf19-ab6b04fcde7d.png)
![image](https://user-images.githubusercontent.com/11186480/187841065-3c6e49ec-9c72-4196-aaa9-901395cb7c9e.png)

Reply:
hmm, you're right. I think we need to reveal parts of information to users, so we can utilize `startingHeight` to adjust block height, especially assertions is empty we need to display no assertions content to tell users.